### PR TITLE
New branch

### PR DESCRIPTION
**Pull request by sebastech**
first pull request - portfolio

### CODE REVIEW

File: portfolio/src/main/webapp/background.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <!--<title>My Portfolio - Sebastian Lopez</title>-->
```
Comment (KaylaNguyen) [inline portfolio/src/main/webapp/background.html]: Is this code commented out? If so, please remove it completely. Same for other files.

File: portfolio/src/main/webapp/background.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        <p> Hi, my name is Sebatian Lopez.
+            I attend the University of Southern California,
+            where I'm majorin in Computer Science!  I'm 
+            originally from Los Angeles, and I enjoy the sunny weather.
+            Some of my hobbies include: playing guitar, playing basketball,
+            and coding things like this portfolio.  I also enjoy sci-fi media,
+            from TV shows to books to audiobooks.  Looking forward to future SPS
+            projects!
+        </p>
+
```
Comment (KaylaNguyen) [inline portfolio/src/main/webapp/background.html]: Remove extra new line. Same for other files.

File: portfolio/src/main/webapp/game.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
+
+
+
+var myVar, canvas, ctx, x, y, dx, dy, factor, color, colorStr;
```
Comment (KaylaNguyen) [inline portfolio/src/main/webapp/game.js]: Global vars are discouraged in js. Can you make sure among these vars, only the one you absolutely need to be global are declared here?

File: portfolio/src/main/webapp/game.js [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+function init() {
+    canvas = document.getElementById("myCanvas");
+    ctx = canvas.getContext("2d");
+    x = canvas.width/2;
+    y = canvas.height-30;
+    dx = 2;
+    dy = -2;
+    factor = 1;
+    myVar = setInterval(draw, 10);
```
Comment (KaylaNguyen) [inline portfolio/src/main/webapp/game.js]: You may want to use a more descriptive variable name for this.

File: portfolio/src/main/webapp/game.js [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+    drawStar(colorStr);
+    if(x + dx > canvas.width-(10*factor) || x + dx < (10*factor)) {
+        dx = -dx;
+    }
+    if(y + dy > canvas.height-(10*factor) || y + dy < (10*factor)) {
+        dy = -dy;
+    }
+    window.addEventListener('keydown', function (event) {
+      if(event.keyCode === 67){
+          //c pressed
```
Comment (KaylaNguyen) [inline portfolio/src/main/webapp/game.js]: Proper comment starts with a full sentence. E.g. // When users press "c" key.

File: portfolio/src/main/webapp/game.js [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+          colorStr = color[Math.floor(Math.random() * 7)];
+      }else if(event.keyCode === 83){
```
Comment (KaylaNguyen) [inline portfolio/src/main/webapp/game.js]: Add space after bracket.

Comment (KaylaNguyen) [reply portfolio/src/main/webapp/game.js]: You may want to add a comment here too for the keycode == 83.

File: portfolio/src/main/webapp/game.js [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+    })
+    x += dx;
+    y += dy;
+}
+
+function drawStar(col) {
+    console.log("in draw star");
+    ctx.fillStyle = col;
+    ctx.beginPath();
+    //ctx.moveTo(108, 0.0);
```
Comment (KaylaNguyen) [inline portfolio/src/main/webapp/game.js]: Remove all comments that don't provide informational values.

File: portfolio/src/main/webapp/script.js [hunk 8]
```diff
@@ -17,7 +17,7 @@
  */
 function addRandomGreeting() {
   const greetings =
```
Comment (KaylaNguyen) [inline portfolio/src/main/webapp/script.js]: If you don't use this function, you can remove it.

File: portfolio/src/main/webapp/style.css [hunk 9]
```diff
@@ -7,3 +7,5 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+canvas { background: #ffd300; }
```
Comment (KaylaNguyen) [inline portfolio/src/main/webapp/style.css:11]: What is this for?

**Review by KaylaNguyen — COMMENTED** [review]
Overall looks good! A few comments about style.
This PR is pretty big. Might be better to split into smaller PRs next time :)

File: portfolio/src/main/webapp/style.css (hunk 9 above)
Comment (sebastech) [reply portfolio/src/main/webapp/style.css:11]: it makes the canvas background yellow

File: portfolio/src/main/webapp/background.html [hunk 10]
```diff
@@ -0,0 +1,21 @@
...
+            where I'm majorin in Computer Science!  I'm 
+            originally from Los Angeles, and I enjoy the sunny weather.
+            Some of my hobbies include: playing guitar, playing basketball,
+            and coding things like this portfolio.  I also enjoy sci-fi media,
+            from TV shows to books to audiobooks.  Looking forward to future SPS
+            projects!
+        </p>
+    </div>
+  </body>
+</html>
```
Comment (KaylaNguyen) [inline portfolio/src/main/webapp/background.html]: Add new lines at end of html files. Same for other files.

File: portfolio/src/main/webapp/game.js [hunk 11]
```diff
@@ -0,0 +1,58 @@
...
+    drawStar(colorStr);
+    if(x + dx > canvas.width-(10*factor) || x + dx < (10*factor)) {
+        dx = -dx;
+    }
+    if(y + dy > canvas.height-(10*factor) || y + dy < (10*factor)) {
+        dy = -dy;
+    }
+    window.addEventListener('keydown', function (event) {
+      if(event.keyCode === 67) {
+          //When users press "c" key.
```
Comment (KaylaNguyen) [inline portfolio/src/main/webapp/game.js]: nit: add space between // and comment.

File: portfolio/src/main/webapp/game.js [hunk 12]
```diff
@@ -0,0 +1,58 @@
...
+          //When users press "s" key.
+           factor = Math.random();
+      }
+    })
+    x += dx;
+    y += dy;
+}
+
+function drawStar(col) {
+    console.log("in draw star");
```
Comment (KaylaNguyen) [inline portfolio/src/main/webapp/game.js]: Remove logging code.

**Review by KaylaNguyen — COMMENTED** [review]
Looks good!

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 13]
```diff
@@ -27,6 +27,6 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    response.getWriter().println("<h1>Hello Sebastian!</h1>");
```
Comment (KaylaNguyen) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Is this part of assignment 2? If so, can we make this part of a different PR?

Comment (KaylaNguyen) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Reopen this thread.
Can you remove this file out of this PR? Once you do, this PR will be purely for assignment 1 and good to submit :)

**Review by KaylaNguyen — CHANGES_REQUESTED** [review]

**Review by KaylaNguyen — COMMENTED** [review]
I think you want to "uncommit" instead of remove the file. If you commit this deleted file, the file in the master branch will also be deleted.
You can follow steps 1-3 in the 1st answer here https://stackoverflow.com/questions/18357511/git-remove-committed-file-after-push

**Review by KaylaNguyen — APPROVED** [review]

**Pull request merged**